platform/broadcom/sonic-platform-modules-accton/as9716-32d/modules/pddf_custom_psu.c

```diff
@@ -9,10 +9,99 @@
 #include <linux/sysfs.h>
 #include <linux/slab.h>
 #include <linux/dmi.h>
-#include "../../../../pddf/i2c/modules/include/pddf_psu_defs.h"
+#include "pddf_psu_defs.h"
 
 ssize_t pddf_show_custom_psu_v_out(struct device *dev, struct device_attribute *da, char *buf);
 extern PSU_SYSFS_ATTR_DATA access_psu_v_out;
+ssize_t pddf_get_custom_psu_model_name(struct device *dev, struct device_attribute *da, char *buf);
```

Comment:
Are various types of PSUs applicable here for AS9716 platform?

Reply:
Current as9716 support ACBEL and YESM1300AM PSU. So other psu code can be moved. I will update code.

Reply:
Done to  remove not needed psu code.